src/test/java/CreatePlayList.java

```diff
@@ -0,0 +1,136 @@
+import Auto.pages.AutoHomePage;
+import Auto.pages.AutoLoginPage;
+import Auto.pages.AutoPlaylistPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CreatePlayList extends BaseTest1{
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. 
2. Use whole words and must avoid acronyms and abbreviations.

Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`extends BaseTest`

Reply:
I refactored class CreatePlaylist with PlaylistTests and BaseTest1 with BaseTest.